kolena/_api/v1/event.py

```diff
@@ -54,6 +54,15 @@ class Event(str, Enum):
         # test run
         EXECUTE_TEST_RUN = "sdk-test-run-executed"
 
+        # dataset
+        REGISTER_DATASET = "sdk-dataset-registered"
+        FETCH_DATASET = "sdk-dataset-fetched"
+        FETCH_DATASET_HISTORY = "sdk-dataset-history-fetched"
+
+        # dataset evaluation
+        FETCH_DATASET_MODEL_RESULT = "sdk-dataset-model-result-fetched"
+        TEST_DATASET_MODEL = "sdk-dataset-model-tested"
```

Comment:
cc @sandalns , can you also help updating naming+wording of this in your [PR](https://github.com/kolenaIO/kolena/pull/390)?

Reply:
Will update in my PR